### ISSUE TITLE
[A11y] AutoSuggestBox "Close" or "Search" button not accessible through keyboard

### ISSUE THREAD

**Issue by alcail** [microsoft/microsoft-ui-xaml#7733]
### Describe the bug

When typing in AutoSuggestBox, "Close" and "Search" button are not accessible through keyboard

### Steps to reproduce the bug

1. Open File Explorer, for example
2. press Tab key till you land on the Search box
3. start typing
4a. press Tab and you lands on the left nav pane
4b. press Arrow keys and nothing is happening

### Expected behavior

you expect pressing Tab or Arrow keys in step 4 to land on the Search buttons - "Close" (red rectangle in screenshot below) and "Search" (orange rectangle in screenshot below)

### Screenshots

![image](https://user-images.githubusercontent.com/82416644/190474562-3751b1fc-70e1-4bcc-aaf0-b205e45ded59.png)

### Windows app type

- [X] UWP
- [ ] Win32

### Device form factor

Desktop

### Windows version

Windows 11 (21H2): Build 22000

**Comment by pratikone** [comment]
which File Explorer is it ? win32 + system xaml island or winui 3 file explorer ?

**Comment by alcail** [comment]
I'm working on a UWP app using WinUI2.

I'm seeing the same issue on FE, but not sure about the its version though.
![image](https://user-images.githubusercontent.com/82416644/195225094-2f4a7f19-7b1d-4496-9d8e-d74ec0cf6c55.png)

I'm on Windows 11
![image](https://user-images.githubusercontent.com/82416644/195225248-1dccb875-dc8b-4657-9953-a36cae88e952.png)

**Comment by alcail** [comment]
I've got some explanation for those buttons.

An internal bug "WinUI 3 Controls Gallery->Auto suggest box: Clear search/Search controls are not accessible with keyboard" was closed with the following explanation:
- This is by design. 
-  Those buttons are accessible only via shortcuts, so no Tab ⁠key: 
   - Clear is CTRL + A to select all, then Delete 
   - Search is only Enter
